policies/secure-development-lifecycle.md

```diff
@@ -0,0 +1,45 @@
+# Secure Development Lifecycle Policy
+
+## Purpose
+The secure development lifecycle describes the approach that is used to define, develop, deliver and maintain software. 
+
+## Scope
+This lifecycle is intended to be used by software teams including product managers, designers, developers and quality assurance to develop secure software products. This policy describes the lifecycle of feature development.
+
+## Policy
+This lifecycle integrates security into all aspects of application software development which is comprised of six phases. 
+
+1. Requirements Gathering and Planning:
+  - Identify the scope, goals, and necessary resources for development
+  - Define functional and security requirements for the feature 
+    - Considering core engineering principles (link)
+    - Security controls such as authentication, data encryption, and secure communication protocols
+
```

Comment:
@hesterbruikman @kevinghim after the decision with Aaron Munda earlier today, do you think we shall include such a sentence?
````
Inform the marketing team of the projected timelines, features, and any potential market impacts
````

Reply:
I think that makes sense. I would slightly adjust:

Inform the marketing team _and customer support_ of the projected timelines, features, and any potential market impacts _such that they can monitor for reports of security incidents_

Reply:
@hesterbruikman my only concern with adding "such that they can monitor for reports of security incidents" is that now these groups would have to show evidence of monitoring. I think we can add the statement without the text quoted in this response. WDYT?